Parsers/ASimAuthentication/Tests/Cisco_Ise_Authentication_DataTest.csv

```diff
@@ -0,0 +1,10 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [EventProduct] of type [Enumerated]: [""ISE""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [EventVendor] of type [Enumerated]: [""Cisco""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [TargetUsernameType] of type [Enumerated]: [""Regular""] (Schema:Authentication)"

```

Comment:
Please provide a valid TargetUsernameType using the schema.
https://learn.microsoft.com/en-us/azure/sentinel/normalization-about-schemas#the-user-entity

Reply:
fixed